team-props/static-analysis/detekt-config.yml

```diff
@@ -74,14 +76,15 @@ complexity:
   active: true
   ComplexCondition:
     active: true
-    threshold: 3
+    threshold: 4
```

Comment:
where is this coming from?

Reply:
Just merging in the defaults, if the thresholds are higher than what we have I raised to match